### DISCUSSION TITLE
Draw功能如何让绘制出来的标记拥有自己的色彩样式呢

### DISCUSSION THREAD

**Discussion by ChauPha** [zouyaoji/vue-cesium#176] [Q&A] [answered]
Draw功能如何让绘制出来的每个标记拥有自己的色彩样式呢
![1648802630(1)](https://user-images.githubusercontent.com/24471468/161228476-66d2848c-95c0-4cc0-abe5-4cf9ebc9b40c.jpg)

**Comment by zouyaoji** [comment, marked as answer]
参考这个 issue。

https://github.com/zouyaoji/vue-cesium/issues/167
